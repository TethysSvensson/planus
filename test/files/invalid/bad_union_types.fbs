union A0 {
  A: A0,
  B: A1,
  C: A2,
  D: [bool],
  E: [bool: 1],
  F: bool,
  G: uint8,
  H: float32,
}

struct A1 { x: bool; }
enum A2: uint8 {}
