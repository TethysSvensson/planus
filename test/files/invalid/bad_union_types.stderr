error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:2:6
  │
2 │   A: A0,
  │      ^^ Unions in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:3:6
  │
3 │   B: A1,
  │      ^^ Structs in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:4:6
  │
4 │   C: A2,
  │      ^^ Enums in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:5:6
  │
5 │   D: [bool],
  │      ^^^^^^ Vectors in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:6:6
  │
6 │   E: [bool: 1],
  │      ^^^^^^^^^ Arrays in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:7:6
  │
7 │   F: bool,
  │      ^^^^ Bools in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:8:6
  │
8 │   G: uint8,
  │      ^^^^^ Integers in unions are not supported

error: Unsupported type
  ┌─ test/files/invalid/bad_union_types.fbs:9:6
  │
9 │   H: float32,
  │      ^^^^^^^ Floats in unions are not supported

Error: Bailing because of errors
